generate.sh

```diff
@@ -7,7 +7,8 @@ if [ $# -ne 1 ]
 fi
 
 mkdir -p temp
-cp -r configure.js configure.sh install.sh package.json processes.json README.md restart.sh start.sh stop.sh upgrade.sh src wickrio-bot-web temp
+
+cp -r  build configure.js configure.sh install.sh package.json processes.json README.md restart.sh start.sh stop.sh upgrade.sh wickrio-bot-web temp
```

Comment:
the build directory does NOT exist when generate.sh is run.  Generate.sh is run on the build machine, the build machine does NOT install the product before using the generate.sh to create the software.tar.gz file

Reply:
ohhh